docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -11,46 +11,62 @@ other resources such as Kubernetes clusters and applications coming in the futur
 ## Installation & upgrade
 
 Head over to the [Downloads](https://goteleport.com/download/) page to download the most recent
-version of Teleport Connect. As of now, Teleport Connect supports macOS only. Double-click the
-downloaded `.dmg` file and drag the Teleport Connect icon to the Applications folder.
+version. Teleport Connect supports macOS, Linux, and Windows.
+
+<Tabs>
+<TabItem label="macOS">
+Double-click the downloaded `.dmg` file and drag the Teleport Connect icon to the Applications folder.
 
 To upgrade Teleport Connect to a newer version, drag the new version to the Applications folder.
+</TabItem>
+<TabItem label="Linux">
+Download the DEB (Debian-based distros) or RPM (RHEL-based distros) package and install it using
+your package manager. Repeat the process for in-place upgrades.
+
+You can also download the project as a `tar.gz` file to extract and run it in place.
```

Comment:
I think a copyable code snippet for the `tar` command would be good here

https://xkcd.com/1168/

Reply:
You mean like this?

````suggestion
You can also download the project as a `tar.gz` file to extract and run it in place:

```command
$ tar -xf  teleport-(=teleport.version=)-linux-*.tar.gz
```
````